aict_tools/scripts/apply_disp_regressor.py

```diff
@@ -140,6 +144,8 @@ def main(
             key = config.events_key
             append_column_to_hdf5(data_path, source_x, key, "source_x_prediction")
             append_column_to_hdf5(data_path, source_y, key, "source_y_prediction")
+            append_column_to_hdf5(data_path, source_alt, key, "source_alt_prediction")
```

Comment:
if coordinate transforms are `FACT`, we should fill zenith I think.

Reply:
Ok